src/credential/sd_jwt_vc.rs

```diff
@@ -0,0 +1,169 @@
+use base64::prelude::*;
+use ssi::{
+    claims::{
+        jwt::{AnyClaims, JWTClaims},
+        sd_jwt::{RevealedSdJwt, SdJwtBuf},
+        vc::{v1::Credential as _, v2::Credential as _},
+        JwsString,
+    },
+    prelude::AnyJsonCredential,
+};
+use uuid::Uuid;
+
+use crate::{CredentialType, KeyAlias};
+
+use super::VcdmVersion;
+
+#[derive(uniffi::Object, Debug, Clone)]
+/// A verifiable credential secured as a JWT.
```

Comment:
```suggestion
/// A VCDM2.0 verifiable credential secured as an SD-JWT.
```
The [JOSE-COSE SD-JWT VC format](https://www.w3.org/TR/vc-jose-cose/#with-sd-jwt) is only specified for VCDM2.0 credential.

Reply:
No longer have this type with comment. Resolving for now.